### ISSUE TITLE
Subscribers option no longer working with Ghost 3

### ISSUE THREAD

**Issue by SoldierCorp** [eddiesigner/liebling#78]
Hi!

Do you have an alternative to show the newsletter subscription for Ghost 3?

**Comment by Xenonstaforums** [comment]
Yeah. I have the same problem. I hope the developer will push an update as soon as possible.

**Comment by eddiesigner** [comment]
Sorry guys but that feature has been replaced with the Members labs feature: https://ghost.org/faq/upgrades/

There is nothing I can do at the moment since there is no fallback or some variable I can check in order to switch between features.

The thing you can do for now is not using Ghost 3 until they provide a way to create a fallback (if they ever do so).

**Comment by BraydenGirard** [comment]
According to the 3.0 docs you can still have a subscription form with just an email.

https://ghost.org/docs/members/signup-signin-forms/

**Comment by eddiesigner** [comment]
That kind of subscription is different. That subscription form is to let visitors become members who eventually will become paid users. That means that's the first step of the paid subscription flow, not the regular newsletter subscription.

Even if they use it as a newsletter subscription too (I couldn't find info about it), I would need to create a new theme (Liebling for Ghost 3 or something) because the new attributes are no available in Ghost 2 and as I said, there is no fallback or variable that I can check to switch between  features.

**Comment by daviddarnes** [comment]
Hey @eddiesigner 👋. Noticed that people are having problems with the switch from Subscribers to Members in your theme. Members can be used in the same fashion, but for your theme to work with both you'll need to make use of the flag `@labs.members`. This can be used like so:
```handlebars
{{#if @labs.members}}
    <!-- members form here -->
{{/if}}
```
There's an [example in Casper](https://github.com/TryGhost/Casper/blob/ff51925567fdc135132f779fad5e99540439b986/post.hbs#L113-L115).

We'll get this added to our docs properly as you rightfully pointed out it's not there yet. We do have some documentation here on how to switch from Subscribers to Members which might help you cater for both in your theme: https://ghost.org/faq/subscribers-to-members/

If you need some help just ping me! ✌️

**Comment by eddiesigner** [comment]
@daviddarnes Thank you for the info! I'm aware of the flag `@labs.members` but I wasn't sure if that flag could work for both versions, now I know that I can use it 🙂

Anyway, I have plans to work on this theme later this month to give full support for Ghost 3 ;)

**Comment by daviddarnes** [comment]
@eddiesigner awesome! Hit me up if you have any questions 📥 . 

By the way, maybe it's worth putting the releases link in the "Website" field on the repo ([next to this field](https://github.com/eddiesigner/liebling#repo_homepage)). Might prevent issues when people download the repo instead by accident 😅

**Comment by eddiesigner** [comment]
@daviddarnes I will! Thanks for the tip 😉

**Comment by baptiste0928** [comment]
@eddiesigner Any changes about that? It would be really nice if the theme is compatible with members (even if it's already great 😉)!

**Comment by eddiesigner** [comment]
@baptiste0928 I already started working on this, it should be ready in the following days or next week. Although giving full support to member functionality will take more time because it involves designing and developing several more screens and I don't have much spare time at the moment 😟

**Comment by eddiesigner** [comment]
I just released [version 0.6.6](https://github.com/eddiesigner/liebling/releases/tag/v.0.6.6) which gives support to Ghost 3 and the new subscribers form 📢 

To learn how to enable it please read this: https://github.com/eddiesigner/liebling/wiki/How-to-enable-subscribers

_Note: From now on, this theme is only compatible with Ghost 3+_
